packages/runner/injection/cross-origin.js

```diff
@@ -25,7 +25,7 @@ const findCypress = () => {
         // eslint-disable-next-line no-useless-concat
         const frameHostRegex = new RegExp(`(^|\\.)${ frame.location.host.replaceAll('.', '\\.') }` + '$')
 
-        // Compare the locations origin policy without pulling in more dependencies.
+        // Compare the locations origins without pulling in more dependencies.
         // Compare host, protocol and test that the window's host ends with the frame's host.
         // This works because the spec bridge's host is always created without a sub domain.
         if (window.location.port === frame.location.port
```

Comment:
We should be able to do to `window.location.origin` === `frame.location.origin` here